Formula/rure.rb

```diff
@@ -0,0 +1,32 @@
+class Rure < Formula
+  desc "C API for RUst's REgex engine"
+  homepage "https://github.com/rust-lang/regex/tree/master/regex-capi"
+  url "https://github.com/rust-lang/regex/archive/1.5.4.tar.gz"
+  version "0.2.1"
```

Comment:
Yeah I was surprised at this too. The C library has a separate versioning scheme from the Rust library that the source is distributed with (C library version defined [here](https://github.com/rust-lang/regex/blob/master/regex-capi/Cargo.toml)).

I think what's going on here is that the C library version gets bumped when there are changes to the actual interface code, but releases for the Rust code that the C library wraps happen more frequently and don't cause the C library version to get bumped. You can see that upstream has a `regex-capi-0.2.1` tag [here](https://github.com/rust-lang/regex/tree/regex-capi-0.2.1), but this tag doesn't contain any of the updates to the Rust code that have happened between March 2019 and now.

Any suggestions here? I'm sort of inclined to just change this to look at the version of the Rust library and ignore the versioning scheme in the regex-capi Cargo.toml file entirely; I can't think of a reason that a consumer of this library would actually want "version 0.2.1 of the C library but without any of the changes to the Rust code that have taken place since that was released", which is what the `regex-capi-0.2.1` tag provides.

Reply:
I think we should just build from the `regex-capi-0.2.1` tag and make sure `livecheck` looks only for `regex-capi-*` tags. This should explain how to do that: https://docs.brew.sh/Brew-Livecheck there are also many examples in this repo.

Reply:
Yes, let's just version this as `1.5.4` then. Keeping this pinned to the `capi` version means that users don't get updates when there are updates to the internals.

Reply:
Great, I'll update it to say that this is `1.5.4`. Since that version number will be in the GitHub tag I don't need to do any custom livecheck stuff, right?

Reply:
Yes.